pages/tutors.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <main class="cover">
+    <div class="cover_container">
+      <div class="head">
+        <h4>HUB <span>TUTORES</span></h4>
+        <p>
+          Um dos principais piláres da nossa comunidade é a Ajuda! Para isso, nós “qualificamos” alguns membros da comunidade que se colocam à disposição de ajudar outros membros, conforme seu nível de conhecimento em certas áreas.
+        </p>
+        <input type="text" name="search" class="search" placeholder="Pesquisar">
+      </div>
+      <div class="content">
+        <div class="grid">
+          <card-instructor
+            v-for="(stack, index) in stacks" :key="index"
+            :stack="stack"
```

Comment:
acredito que aqui o nome pode ser mudado...
stacks pq ? 
ideal seria `tutor in tutores`

Vc nao esta criando card de stacks... mas de Tutores ;)

Reply:
Mas a prop que está sendo usada é uma stack, então não teria sentido eu por outra coisa